src/test/java/com/google/sps/CreateGameTest.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.utils.GameUtils;
+
+/** tests the createGameEntity function */
+@RunWith(JUnit4.class)
+public final class CreateGameTest {
+  
+  private GameUtils gameUtils;
+  private DatastoreService datastore;
+
+  // helper variable allows the use of datastore in testing 
+  private final LocalServiceTestHelper helper =
+    new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    gameUtils = new GameUtils();
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Given an empty string for game name, the function should return false
+  @Test
+  public void emptyGameName() {
```

Comment:
My preference is to include the tested behavior in the name of the test method, e.g. `addingEmptyGameNameFails()`. (This is just a personal thing, though. I think Devon disagrees with me. Do whatever you want.)

Reply:
I added "fails" to the ones that are supposed to fail